app/connectors/OtherEmploymentIncomeConnector.scala

```diff
@@ -17,24 +17,24 @@
 package connectors
 
 import config.AppConfig
-import connectors.parsers.OtherEmploymentsIncomeHttpParser.{OtherEmploymentsIncomeHttpReads, OtherEmploymentsIncomeResponse}
+import connectors.parsers.OtherEmploymentIncomeHttpParser.{OtherEmploymentIncomeHttpReads, OtherEmploymentIncomeResponse}
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
 import utils.DESTaxYearHelper.desTaxYearConverter
 
 import java.net.URL
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class GetOtherEmploymentIncomeConnector @Inject()(val http: HttpClient,
-                                                  val appConfig: AppConfig)(implicit ec: ExecutionContext) extends DesConnector {
+class OtherEmploymentIncomeConnector @Inject()(val http: HttpClient,
```

Comment:
Why remove the `Get`? I am thinking now if we should remove the Get/create and add the methods inside this same connector

Reply:
tech debt task created here: https://jira.tools.tax.service.gov.uk/browse/SASS-5147